v63004/u_inref/gtm8874.csh

```diff
@@ -33,6 +37,10 @@ $ydb_dist/mumps -run testA^gtm8874
 
 echo '# Shut down the DB and backup necessary files to sub directory'
 $gtm_tst/com/dbcheck.csh >>& dbcreate_log_1.txt
```

Comment:
Better to have separate log files for dbcreate and dbcheck i.e. dbcreate_log_1 -> dbcheck_log_1

Similar change needed in a few other places.